topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.bib

```diff
@@ -0,0 +1,30 @@
+
+# This is the bibliography file for your tutorial.
+#
+# To add bibliography (bibtex) entries here, follow these steps:
+#  1) Find the DOI for the article you want to cite
+#  2) Go to https://doi2bib.org and fill in the DOI
+#  3) Copy the resulting bibtex entry into this file
+#
+# To cite the example below, in your tutorial.md file
+# use {% Batut2018 %}
+
+@article{Batut2018,
```

Comment:
Please remove this unused citation

Reply:
Removed

Reply:
Both the tutorials have been changed according to the suggestions.